.github/workflows/ci.yml

```diff
@@ -65,32 +67,28 @@ jobs:
         run: |
           echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
 
-      # These are pulled in order to be able to use docker layer caching
-      - name: Pull snuba CI images
-        if: github.repository_owner == 'getsentry'
-        run: |
-          docker pull ghcr.io/getsentry/snuba-ci:${{ github.sha }} || \
-            docker pull ghcr.io/getsentry/snuba-ci:${{ steps.branch.outputs.branch }} || \
-            docker pull ghcr.io/getsentry/snuba-ci:latest || true
-
       - name: Build snuba docker image for CI
         if: github.repository_owner == 'getsentry'
         run: |
           docker build . \
-            -t ghcr.io/getsentry/snuba-ci:latest \
-            -t ghcr.io/getsentry/snuba-ci:${{ github.sha }} \
-            -t ghcr.io/getsentry/snuba-ci:${{ steps.branch.outputs.branch }} \
-            --cache-from ghcr.io/getsentry/snuba-ci:latest \
-            --cache-from ghcr.io/getsentry/snuba-ci:${{ steps.branch.outputs.branch }} \
-            --cache-from ghcr.io/getsentry/snuba-ci:${{ github.sha }} \
+            --cache-from $SNUBA_IMAGE:latest \
+            --cache-from $SNUBA_IMAGE:${{ steps.branch.outputs.branch }} \
+            -t $SNUBA_IMAGE:latest \
+            -t $SNUBA_IMAGE:${{ steps.branch.outputs.branch }} \
+            -t $SNUBA_IMAGE:${{ github.sha }} \
```

Comment:
`github.sha` for a PR will be the commit sha of the merge commit, however we generally squash our commits to master so on master, it will be a different sha.